src/main/java/org/broadinstitute/consent/http/resources/TDRResource.java

```diff
@@ -85,59 +70,4 @@ public Response getDatasetByIdentifier(@Auth AuthUser authUser,
     }
   }
 
-  @POST
```

Comment:
This is the work @rushtong mentioned a new ticket for in standup.

Reply:
This covers https://broadworkbench.atlassian.net/browse/DT-1523